chainstate/tx-verifier/src/transaction_verifier/mod.rs

```diff
@@ -45,6 +45,7 @@ pub use input_output_policy::IOPolicyError;
 
 use std::collections::BTreeMap;
 
+pub use self::transferred_amount_check::calculate_required_fee_burn;
```

Comment:
Please note that fee burn calculations have changed after the fork, and also keep in mind that Gosha is moving fee calculations for block rewards to the accumulator.

Reply:
fixed with the usage of the ConstrainedValueAccumulator